docs/sources/operator/architecture.md

```diff
@@ -43,6 +43,24 @@ A single resource can belong to multiple hierarchies. For example, if two
 GrafanaAgents use the same Probe, modifying that Probe will cause both
 GrafanaAgents to be reconciled.
 
+### Build the Hierarchy
+
+The hierarchy is built via label matching on the custom resources. Below is a
+graph illustrating the matching. The `GrafanaAgent` picks up the `MetricsInstance`
+and `LogsInstance` that match the labels `app.kubernetes.io/name: loki` and `app.kubernetes.io/instance: release-name`. The instances pick up the resources the same way.
+
+<img src="../../assets/hierarchy.svg" alt="Hierarchy" />
```

Comment:
The text on the edges in this graph is always the same and it makes the image busy.  Should we leave it out since you say this in the text above the image?  Another approach might be to add a callout on the left that points to some of the edges that says:

"The operator matches each instance against:

```matchLabels
   app.kubernetes.io/name: loki
   app.kubernetes.io/instance: release-name
```

Reply:
I can work on it later because I'm working on these topics anyways.  It was created by a former writer, Karen M (not to be confused with me, Karen G. 😄 ) .

Reply:
I am also confused about the match labels. The diagram just has `name:` not the full `app.kubernetes.io/name:`. I'd probably update the diagram with the full names?